### ISSUE TITLE
Content files not copied from nupkg when Forms is installed

### ISSUE THREAD

**Issue by stefanolsen** [screentek/Optimizely#3]
Followed the installation instructions, but no content files were copied to the wwwroot folder.

Upon inspecting the Imageshop.Optimizely.Plugin.targets file I found that the copying happens in a custom FormsCopyModuleFiles task. And inspecting the detailed MSBuild log, I noticed these two log entries:

```
Overriding target "FormsCopyModuleFiles" in project "C:\Users\Stefan Holm Olsen\.nuget\packages\imageshop.optimizely.plugin\1.0.40\build\net6.0\Imageshop.Optimizely.Plugin.targets" with target "FormsCopyModuleFiles" from project "C:\Users\Stefan Holm Olsen\.nuget\packages\episerver.forms.ui\5.5.1\build\net6.0\EPiServer.Forms.UI.targets".
Overriding target "FormsCopyModuleFiles" in project "C:\Users\Stefan Holm Olsen\.nuget\packages\episerver.forms.ui\5.5.1\build\net6.0\EPiServer.Forms.UI.targets" with target "FormsCopyModuleFiles" from project "C:\Users\Stefan Holm Olsen\.nuget\packages\episerver.forms\5.5.1\build\net6.0\EPiServer.Forms.targets".
```

It looks to me like the plugin is not working when Episerver.Forms.UI is also installed.

**Comment by stefanolsen** [comment]
Besides, I would prefer to have these ClientResource files in the _modules folder, so they are completely unavailable from outside the UI.

**Comment by respiant** [comment]
Hi, thanks for letting us know. I will look into this and keep you updated when a fix is ready.

**Comment by respiant** [comment]
@stefanolsen 

Hi, I want to let you know that we have solved the issue regarding files not being copied over when EPiServer.Forms.UI was installed. This is solved in version 1.1.1: https://www.nuget.org/packages/Imageshop.Optimizely.Plugin/1.1.1

Regarding your second comment, I've created a new issue. I'll keep it updated when we have solved this as well: https://github.com/screentek/Optimizely/issues/9
